client/src/main/java/com/vesoft/nebula/graph/client/GraphClientImpl.java

```diff
@@ -200,13 +198,12 @@ public ResultSet executeQuery(String statement) throws ConnectionException,
     }
 
     private boolean checkTransportOpened(TTransport transport) {
-        return !Objects.isNull(transport) && transport.isOpen();
+        return !(transport == null) && transport.isOpen();
```

Comment:
`!(transport == null)` that is `transport != null`

Reply:
Smart Catch ！